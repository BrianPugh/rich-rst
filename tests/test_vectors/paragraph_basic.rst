A paragraph must be
left-aligned.

This is a different paragraph.
