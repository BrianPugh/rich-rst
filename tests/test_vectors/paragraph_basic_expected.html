<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<style>
.r1 {color: #f8f8f2; text-decoration-color: #f8f8f2; background-color: #282a36}
body {
    color: #f8f8f2;
    background-color: #282a36;
}
</style>
</head>
<body>
    <pre style="font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace"><code style="font-family:inherit"><span class="r1">A paragraph must be left-aligned.</span>

<span class="r1">This is a different paragraph.</span>
</code></pre>
</body>
</html>
